modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/CassandraAssignmentsRepository.java

```diff
@@ -482,9 +482,9 @@ public void assignUsersInBatch(List<Pair<Experiment, Assignment>> assignments, D
      */
     private void incrementCounts(List<Pair<Experiment, Assignment>> assignments, Date date) {
         boolean countUp = true;
-        assignments.forEach(pair -> {
-            assignmentsCountExecutor.execute(new AssignmentCountEnvelope(this, experimentRepository, dbRepository, pair.getLeft(), pair.getRight(), countUp, eventLog, date, assignUserToExport, assignBucketCount));
-        });
+        assignments.forEach(pair -> assignmentsCountExecutor.execute(new AssignmentCountEnvelope(
```

Comment:
Is there any logical change in code here or just formatting? You may want to check the code style formatter...